package.json

```diff
@@ -85,7 +85,7 @@
     "@mnajdova/enzyme-adapter-react-18": "^0.2.0",
     "@mui/icons-material": "^5.10.9",
     "@mui/material": "^5.10.12",
-    "@mui/monorepo": "https://github.com/mui/material-ui.git#master",
+    "@mui/monorepo": "https://github.com/mnajdova/material-ui.git#docs/upgrade-nextjs-13",
```

Comment:
Change to `mui` org once https://github.com/mui/material-ui/pull/35001 is merged